bundles/org.openhab.binding.linuxinput/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
```

Comment:
You have used X11 licensed code in this library. Please mention 3rd party files and licenses here

Reply:
Thanks! I doublechecked everything and the constants of the enums `Type` and `BusType` are actually taken from the linux kernel and GPL2. Is this fine?

Reply:
GPL is complicated. I think it requires us to license all files that are using the file to be GPL as well.

```
Include Original 
Describes whether copies of the original software or instructions to obtain copies must be distributed with the software.

 Disclose Source 
If you distribute this software in an executable, you must make your source code available by providing it alongside your distribution or listing an offer, good for 3 years, to obtain the code.

 Include Copyright 
You must conspicuously include the original copyright as well as a disclaimer of warranty.

 State Changes 
Stating significant changes made to software.

 Include License 
Including the full text of license in modified software.
```
from: https://tldrlegal.com/license/gnu-general-public-license-v2

But that's how it is. libevdev seem to be gpl2 so no way around it.

Reply:
Libevdev itself is X11.
Only the constants are GPL2 and they have the [linux syscall notice](https://spdx.org/licenses/Linux-syscall-note.html).
IMO without the constants the API is not usable and it would not make sense to allow the usage of the API without GPL restrictions and having it forced via the constants.